CHANGELOG.md

```diff
@@ -10,6 +10,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### [Deprecated]
 ### [Removed]
 
+##  [0.3.4] - 2023-09-06
```

Comment:
Is this a duplicate of the previous PR?

Reply:
Nope, this is for staging. I already made the release.